proxy/proxy.go

```diff
@@ -845,10 +877,9 @@ func (p *Proxy) makeBackendRequest(ctx *context, requestContext stdlibcontext.Co
 		return res, nil
 	}
 
-	if endpoint != nil {
-		endpoint.Metrics.IncInflightRequest()
-		defer endpoint.Metrics.DecInflightRequest()
-	}
+	entry := p.registry.GetMetrics(host)
```

Comment:
I think it's better not to think about what backend type it is, but add the data available at this point regardless of backend type at all.

Reply:
Resolved in scope of https://github.com/zalando/skipper/pull/2824